<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProgrammerObserver</class>
 <widget class="QWidget" name="ProgrammerObserver">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ProgrammerObserver</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnResolve">
       <property name="text">
        <string>Resolve</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRemove">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
